src/coinbase/wallet.ts

```diff
@@ -748,6 +749,37 @@ export class Wallet {
     return (await this.getDefaultAddress()).createPayloadSignature(unsignedPayload);
   }
 
+  /**
+   * Creates a Webhook.
+   *
+   * @param notificationUri - [String] The URI to which the webhook notifications will be sent.
+   * @param signatureHeader - [String] (Optional) A header used to sign the webhook request,
+   *   defaulting to an empty string.
+   *
+   * @returns [Coinbase::Webhook] The newly created webhook instance.
```

Comment:
removes types, these are for docs in ruby

Reply:
done